src/power_grid_model_io/converters/gaia_excel_converter.py

```diff
@@ -8,13 +8,13 @@
 from pathlib import Path
 from typing import Optional, Union
 
-from power_grid_model_io.converters.tabular_converter import TabularConverter
+from power_grid_model_io.converters.vision_excel_converter import VisionExcelConverter
 from power_grid_model_io.data_stores.gaia_excel_file_store import GaiaExcelFileStore
 
 DEFAULT_MAPPING_FILE = Path(__file__).parent.parent / "config" / "excel" / "gaia_{language:s}.yaml"
 
 
-class GaiaExcelConverter(TabularConverter):
+class GaiaExcelConverter(VisionExcelConverter):
```

Comment:
Do we want this kind of inheritance? i.e. Base-Tabular-VisionExcel-GaiaExcel

Seems to me that you'd want Vision and Gaia both to be final child classes. It would maybe create a bit more overhead, but it does not seem logical to me to let Gaia inherit from Vision, since the two programs are completely different.

Then you would need an `ExcelConverter` class in between and get:
Base-Tabular-Excel-VisionExcel
and 
Base-Tabular-Excel-GaiaExcel

What do you think?

Reply:
Agreed